BitwardenShared/Core/Platform/Services/Stores/AppSettingsStore.swift

```diff
@@ -786,10 +786,10 @@ extension DefaultAppSettingsStore: AppSettingsStore {
                 key = "passwordGenerationOptions_\(userId)"
             case let .pinProtectedUserKey(userId):
                 key = "pinKeyEncryptedUserKey_\(userId)"
-            case .preAuthEnvironmentUrls:
-                key = "preAuthEnvironmentUrls"
-            case let .accountCreationEnvironmentUrls(email):
-                key = "accountCreationEnvironmentUrls_\(email)"
+            case .preAuthEnvironmentURLs:
+                key = "preAuthEnvironmentURLs"
+            case let .accountCreationEnvironmentURLs(email):
+                key = "accountCreationEnvironmentURLs_\(email)"
```

Comment:
⚠️ Changing this key will cause any existing values stored in UserDefaults with this key to become lost/orphaned without a migration. Is that ok in this scenario or do we need to consider how to handle this?

Reply:
🤦🏼‍♀️ Good catch. We shouldn't be making changes to the actual keys like this. I thought I caught all the cases we would do that, but I guess I missed one. I'll pull this one back.